wire-ios-request-strategy/Sources/Payloads/Processing/PrekeyPayloadProcessor.swift

```diff
@@ -46,18 +46,15 @@ public final class PrekeyPayloadProcessor: PrekeyPayloadProcessorInterface {
         from payload: Payload.PrekeyByQualifiedUserID,
         with selfClient: UserClient,
         context: NSManagedObjectContext
-    ) -> Bool {
```

Comment:
The doc comment still mentions a Bool return value. Also below.

Reply:
Fixed